app/models/minion.rb

```diff
@@ -0,0 +1,4 @@
+# Minion represents the minions that have been registered in this application.
+class Minion < ApplicationRecord
+  default_scope { order hostname: :asc }
```

Comment:
I didn't know about ```unscoped```. Probably because I removed ```default_scope``` from my options and never checked back again :). It might still be an issue if you have other scopes applied (chained) and you only want to remove the default_scope one. That's an edge case so we can leave it as is.